app/src/main/java/cz/cvut/fjfi/pvs/pvs2016/util/Photo.java

```diff
@@ -0,0 +1,65 @@
+package cz.cvut.fjfi.pvs.pvs2016.util;
+
+import java.util.HashSet;
+import java.util.Iterator;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+public class Photo {
+	public static class Series {
+		public String name;
+		public int index;
+
+		public Series(String name, int index) {
+			this.name = name;
+			this.index = index;
+		}
+
+		@Override
+		public boolean equals(Object obj) {
+			if (!(obj instanceof Series))
+				return false;
+			if (obj == this)
+				return true;
+			Series rhs = (Series) obj;
+			return (rhs.index == this.index && rhs.name.equals(this.name));
+		}
+
+		@Override
+		public int hashCode() {
+			int result = 17;
+			result = 31 * result + name.hashCode();
+			result = 31 * result + index;
+			return result;
+		}
+
+	}
+
+	public String id;
+	public String path;
+	public HashSet<String> tags;
+	public HashSet<Series> series;
+	public String timestamp;
```

Comment:
nechceme mit nejak typovanou casovou znacku? Treba java.util.Date? s tim, ze z toho jsonu by se to naparsovalo... jde o to, abychom pak mohli fotky pohodlne radit podle casu a podobne